tests/store/model_registry/test_file_store.py

```diff
@@ -1,1364 +1,1341 @@
-
-        # Error cases
-        with pytest.raises(MlflowException, match="Registered model name cannot be empty."):
-            fs.create_registered_model(None)
-        with pytest.raises(MlflowException, match="Registered model name cannot be empty."):
-            fs.create_registered_model("")
-
-        name = random_str()
-        model = fs.create_registered_model(name)
-        assert model.name == name
-        assert model.latest_versions == []
-        assert model.creation_timestamp == model.last_updated_timestamp
-        assert model.tags == {}
-
-    def _verify_registered_model(self, fs, name):
-        rm = fs.get_registered_model(name)
-        assert rm.name == name
-        assert rm.creation_timestamp == self.rm_data[name]["creation_timestamp"]
-        assert rm.last_updated_timestamp == self.rm_data[name]["last_updated_timestamp"]
-        assert rm.description == self.rm_data[name]["description"]
-        assert rm.latest_versions == self.rm_data[name]["latest_versions"]
-        assert rm.tags == self.rm_data[name]["tags"]
-
-    def test_get_registered_model(self):
-        fs = self.get_store()
-        self._create_registered_models_for_test()
-        for name in self.registered_models:
-            self._verify_registered_model(fs, name)
-
-        # test that fake registered models dont exist.
-        for name in {random_str(25) for _ in range(10)}:
-            with pytest.raises(
-                MlflowException, match=f"Could not find registered model with name {name}"
-            ):
-                fs.get_registered_model(name)
-
-    def test_rename_registered_model(self):
-        fs = self.get_store()
-        self._create_registered_models_for_test()
-        model_name = self.registered_models[random_int(0, len(self.registered_models) - 1)]
-
-        # Error cases
-        with pytest.raises(MlflowException, match="Registered model name cannot be empty."):
-            fs.rename_registered_model(model_name, None)
-        # test that names of existing registered models are checked before renaming
-        other_model_name = None
-        for name in self.registered_models:
-            if name != model_name:
-                other_model_name = name
-                break
-        with pytest.raises(
-            MlflowException, match=rf"Registered Model \(name={other_model_name}\) already exists."
-        ):
-            fs.rename_registered_model(model_name, other_model_name)
+@pytest.fixture(scope="function")
+def test_root(tmp_path_factory):
+    tmp_path = str(tmp_path_factory.mktemp("test_root") / f"test_file_store_{random_int()}")
+    return tmp_path
 
-        new_name = model_name + "!!!"
-        assert model_name != new_name
-        fs.rename_registered_model(model_name, new_name)
-        assert fs.get_registered_model(new_name).name == new_name
 
-    def _extract_names(self, registered_models):
-        return [rm.name for rm in registered_models]
+@pytest.fixture(scope="function")
```

Comment:
Do we need to explicitly declare that the scope is the default value?

Reply:
No, we can remove it.

Reply:
deleted all scopes, resolved in https://github.com/wkrt7/mlflow/commit/bffd29e0461cf99eeac69de975a44c6b8361ace9